repository-data/application/src/main/resources/hcm-config/namespaces/hee/news.yaml

```diff
@@ -266,7 +266,7 @@ definitions:
           /rightHandBlocks:
             jcr:primaryType: frontend:plugin
             caption: Right hand objects
-            compoundList: hee:contactCardWithDescriptionReference,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:QuickLinks
+            compoundList: hee:contactCardWithDescriptionReference,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:internalLinksCardReference,hee:QuickLinks
```

Comment:
Could you add the rendering support for `hee:internalLinksCardReference` on `repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/news-main.ftl`:
```
<#case "uk.nhs.hee.web.beans.InternalLinksCardReference">
    <@hee.internalLinksCard card=block.internalLinksCard/>
    <#break>
```